.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
         alwaysTryTypes: true,
       },
     },
+    'import/core-modules': ['indy-vdr-test-nodejs'],
```

Comment:
We shouldn't need this I think. What is the error you're getting?



```suggestion
```

Reply:
I am getting this no-extranous-dependencies error from lint

I figured out it's from this

```
try {
  require('indy-vdr-test-nodejs')
} catch (error) {
  throw new Error('Error registering nodejs bindings for Indy VDR')
}

```

Reply:
ah you can add an eslint ignore above the import, it's fine in this case

Reply:
This is the full error message

`'indy-vdr-test-nodejs' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies`

Reply:
This should work:

```
try {
  // eslint-disable-next-line import/no-extraneous-dependencies
  require('indy-vdr-test-nodejs')
} catch (error) {
  throw new Error('Error registering nodejs bindings for Indy VDR')
}
```